AUTHORS

```diff
@@ -0,0 +1,78 @@
+Alexander Rose <alexander.rose@weirdbyte.de>
```

Comment:
Woud you mind removing this file? (The same information about contributors should be available through github)